central/reports/scheduler/schedule.go

```diff
@@ -319,7 +351,15 @@ func formatMessage(rc *storage.ReportConfiguration) (string, error) {
 		data.WhichVulns = fmt.Sprintf("for new vulnerabilities since %s",
 			timestamp.FromProtobuf(rc.LastSuccessfulRunTime).GoTime().Format("January 02, 2006"))
 	}
-	tmpl, err := template.New("emailBody").Parse(vulnReportEmailTemplate)
+
+	var brandedVulnReportEmailTemplate string
+	if defaults.ImageFlavorEnv() == defaults.ImageFlavorNameStackRoxIORelease {
```

Comment:
No, image flavor doesn't have much to do with product branding and besides "stackrox io release" is NOT community release. It is commercial release for inherited customers from the time when StackRox was independent company.
That's unfortunate coincidence of "stackrox.io" meaning community project and "stackrox.io" hosting image repositories for inherited commercial customers.

Please grep source for `REACT_APP_ROX_PRODUCT_BRANDING`. That's a **separate** environment variable that defines UI branding.
Branding at the backend should be defined similarly. Please don't conflate branding with image flavor because it will be hard to disentangle the two in case we need to do so in the future (highly likely because the rules have changed already at least once).

Also, for this PR to be considered complete, you should make sure that the environment variable is set in CI. Local builds should produce the results consistent as well: UI branding must match backend branding in all cases.
In other words, make sure that `${repo_root}/Makefile` and `${repo_root}/.circleci/config.yml` files are updated and consistent with each other.

Reply:
Sorry, I haven't seen this comment before sending mine. But at least we're on the same page 🙂

Reply:
What's the purpose of having two separate env variables for product branding the UI and the backend? Rather than add a `BACKEND_ROX_PRODUCT_BRANDING` variable can't I just use the value stored in `REACT_APP_ROX_PRODUCT_BRANDING`? Since UI branding must match backend branding in all cases why not have one env variable for both?

Reply:
I'm not saying that the backend should introduce another environment variable.
React forces environment variables to begin with `REACT_APP_`. That's their convention, otherwise the variable is not seen.
Either you will have to make the backend respect something that begins with `REACT_APP_` or look for other creative way. That's your call.

Reply:
Just had a discussion with Fred and it does seem to make the most sense to solve this via a new environment variable. I've also pushed code that effects this change. What's still missing is integrating the new environment variable into circleci but since this seems to be non-trivial Fred and I agreed to create a followup task for that instead, and to leave it out of this PR so it can be resolved quicker. I hope that seems reasonable to you.

Since I have implemented all other ideas from feedback I'll ask for re-review now.

Reply:
Ok. Let's try the way you suggest.